.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: 🐛 Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in pyhf!**
+
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+
+      [issue search]: https://github.com/scikit-hep/pyhf/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain the problem briefly below.
+    placeholder: >-
+      When I try to do X with pyhf and the following workspace, Y breaks or
+      Z happens in an unexpected manner.
+      Here are all the details I know about this problem.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: pyhf Version
+    description: >-
+      Paste verbatim output from `pyhf --version` below, under
+      the prompt line. Please don't wrap it with tripple backticks &mdash; your
+      whole input will be turned into a code snippet automatically.
+    render: console
+    placeholder: |
+      $ pyhf --version
+      pyhf, version 0.6.2
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS / Environment
+    description: >-
+      Provide the name of your operating system and the output of `uname --all`.
+    placeholder: |
+      Ubuntu 18.04
+      $ uname --all
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Steps to Reproduce
+    description: >-
+      Describe exactly how to reproduce the problem, using a minimal test-case.
+      It would *really* help us understand your problem if you paste in the Python code
+      that you're running.
+
+
+      **HINT:** You can paste [GitHub Gist](https://gist.github.com) links for larger files.
+    value: |
+      <!--- Paste your minimal failing Python example code between the quotes below -->
```

Comment:
Maybe easier to use if the comment is above the cell, and the whole cell is `python` formatted? Not clear to me why this different style is used here.

Reply:
Probably to give people a chance to also add text, not just code.

Reply:
Good point. I assumed extra text could go into summary / expected results / actual results, but for more complicated setups the opportunity for more text seems useful.

Reply:
You could have two separate fields for code and text, of course. But then, you can never guess how many code blocks the reporter would need.

Reply:
Good idea. I should probably add additional optional entries for reporting CLI API use bugs and for uploading workspaces.

Reply:
> You could have two separate fields for code and text, of course. But then, you can never guess how many code blocks the reporter would need.

Yeah. I think I may stick with doing the hint and then just adding an optional place for people to upload a workspace file. This way I don't have to have both an entry for CLI and for Python APIs.